.github/workflows/scripts/license.py

```diff
@@ -0,0 +1,184 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC and other
+# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
```suggestion
#!/bin/env python3
#
# Copyright 2023 Lawrence Livermore National Security, LLC and other
# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
#
# SPDX-License-Identifier: Apache-2.0
```

I'd add a shbang line here and modify the name of the file from `license.py` to `license`. Super minor but I'd also go with `.github/workflows/bin/` instead of `scripts/` just to capitalize on well known standards.

Reply:
File renamed and scripts renamed to bin, thanks @alecbcs!